### DISCUSSION TITLE
Der Zähler funktioniert nicht mehr. WEB öffnet nicht mehr

### DISCUSSION THREAD

**Discussion by Robbi12** [jomjol/AI-on-the-edge-device#3067] [Q&A]
Hallo. Bis vor einer Woche funktionierte es noch ohne Probleme. Nun komme ich nicht mehr auf den Zähler. Keine Verbindung zur Website.
WLAN Verbindung steht und ist gut.
Alles schon neu installiert und dies mehrmals. Auch geflasht mit dem WebFlash.

Jemand eine Idee?

**Comment by caco3** [comment]
Liefere doch bitte mal ein DEBUG log. Du kannst dafür die Web Konsole verwenden: https://jomjol.github.io/AI-on-the-edge-device/
